src/NextGen/Gateways/NextGenTestGateway/nextGenTestGateway.tsx

```diff
@@ -1,19 +1,23 @@
 import {__} from '@wordpress/i18n';
+import type {FormData, Gateway} from '@givewp/forms/types';
 
-window.givewp.gateways.register({
+const gateway: Gateway = {
     id: 'test-gateway-next-gen',
-    beforeCreatePayment(values) {
+    label: 'Test Gateway',
```

Comment:
As far as the Gateway _type_ is concerned, we want it to be aware that it has a label.  Although, the gateway label needs to be set from the server (as we're doing in the gateway registrar) because it can be customized in the backend. So, we don't want gateways to think they can hard code a label here since it won't actually do anything.  What do you think we should do?

Reply:
Resolved per our discussion in 4a5b7ea